test/services/teams/helper/helper.hook.js

```diff
@@ -51,26 +51,39 @@ const patchParams = (params, account = {}) => {
 	params.query = params.query || {};
 	params.authenticated = params.authenticated || true;
 	params.headers = params.headers || {};
-	params.payload = params.payload || { accountId: account._id || '<accountId>' };
+	params.payload = params.payload || {
+		accountId: account._id || '<accountId>',
+	};
 	params.headers = params.headers || _DefaultHeaderParams;
 	params.account = params.account || account;
 	return params;
 };
 
 /** * feathers is hook test ** */
-const _typeof = typeof Symbol === 'function' && typeof Symbol.iterator === 'symbol'
-	? function (obj) {
-		return typeof obj;
-	}
-	: function (obj) {
-		return obj && typeof Symbol === 'function' && obj.constructor === Symbol && obj !== Symbol.prototype ? 'symbol' : typeof obj;
-	};
+const _typeof =
```

Comment:
Codacy found an issue: [Unexpected dangling '_' in '_typeof'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5987920)